docs/v0.2.0-beta.7/quickstart.md

```diff
@@ -26,15 +26,15 @@ export default function() {
 
 Now whenever your Ember app makes a GET request to `/api/authors`, Mirage will respond with this data.
 
-## Dynamic mocks
+## Dynamic data
 
-This works, and is traditionally how HTTP mocking is done - but hard-coded responses like this have a few problems:
+This works, and is a common way to isolate tests from a real HTTP server - but hard-coded responses like this have a few problems:
```

Comment:
"...and is a common way to simulate HTTP responses - but hard-coded..."

Since we're not talking about tests here yet I'd rather leave the concept of tests, isolation etc. out, and just focus on the differences between hand-rolling a fake server and using Mirage